music21/musicxml/xmlToM21.py

```diff
@@ -2779,10 +2791,17 @@ def xmlToSimpleNote(self, mxNote, freeSpanners=True):
         # TODO: beams over rests?
         '''
         d = self.xmlToDuration(mxNote)
-        n = note.Note(duration=d)
 
-        # send whole note since accidental display not in <pitch>
-        self.xmlToPitch(mxNote, n.pitch)
+        n = None
+
+        mxUnpitched = mxNote.find('unpitched')
+        if mxUnpitched is None:
+            # send whole note since accidental display not in <pitch>
```

Comment:
should change that to "send entire note" since I didn't understand what wholenotes had to do with it.